tests/test_application.py

```diff
@@ -1287,6 +1287,24 @@ async def callback_2():
             await asyncio.sleep(0.05)
             assert stop_task.done()
 
+    async def test_create_task_awaiting_future(self, app):
+        async def callback():
+            await asyncio.sleep(0.01)
+            return 42
+
+        out = await app.create_task(asyncio.gather(callback()))
```

Comment:
```suggestion
        # `asyncio.gather` returns an `asyncio.Future` and not an
        # `asyncio.Task`
        out = await app.create_task(asyncio.gather(callback()))
```

A reminder for future devs maybe is useful :)